src/lesson06/Daisy.java

```diff
@@ -0,0 +1,12 @@
+package lesson06;
```

Comment:
Let's use as the root package com.flowers.kiev.
Then put all flower-class into the package com.flowers.kiev.entity or com.flowers.kiev.domain
package lesson06;-> package com.flowers.kiev.entity;